Automattic/delphin
# Framework: Use Gridicons package

Gridicons react component was published to npm as its own package. See https://github.com/Automattic/gridicons/pull/188 and https://www.npmjs.com/package/gridicons.

It was already integrated with Calypso by @gwwar in https://github.com/Automattic/wp-calypso/pull/10721. Now it's time to bring it to Delphin. It is replacing `@automattic/dops-components` which was included as a dependency just to take advantage of `Gridicons`.

### Testing
1. Make sure [Credits.md](https://github.com/Automattic/delphin/blob/dbc524b7c34f99af5d589d4e62547dea425452b1/CREDITS.md) got updated with license info for `Gridicons`.
2. Execute `yarn start`.
3. Open http://delphin.localhost:1337 and make sure it loads properly.
4. Browser pages and make sure all icons are there. Some of place to look at:

- Language picker in the footer 
![screen shot 2017-02-02 at 13 20 46](https://cloud.githubusercontent.com/assets/699132/22549217/844b9c9a-e94a-11e6-8a18-d5b0e7d83faf.png)
-  Cross icon for keyword on search page 
![screen shot 2017-02-02 at 13 21 59](https://cloud.githubusercontent.com/assets/699132/22549258/bff45ee4-e94a-11e6-9343-208472f5a7da.png)
- Ellipsis in show me more button on search page 
![screen shot 2017-02-02 at 13 23 39](https://cloud.githubusercontent.com/assets/699132/22549283/e42158ee-e94a-11e6-8b09-e6b0a5030d36.png)


### Review
